draft-ietf-rtcweb-jsep.xml

```diff
@@ -1095,6 +1095,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           described in
           <xref target="sec.addTransceiver" />.</t>
         </section>
+        <section title="removeTrack" anchor="sec.removeTrack">
+
+          <t>The removeTrack method removes a MediaStreamTrack from the
+          PeerConnection, using the RTCRTpSender argument to indicate
+          which track is removed. The sender's track attribute is set to
```

Comment:
It's not obvious why you pass in a RtpSender here - perhaps we should say "which sender should have its track removed".

Also, we haven't discussed the track attribute in this document, so this should probably say "The sender's track is cleared, and the sender stops sending."